katharsis-rs/src/main/java/io/katharsis/rs/JsonApiResponseFilter.java

```diff
@@ -38,6 +40,13 @@ public JsonApiResponseFilter(KatharsisFeature feature) {
 	public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) throws IOException {
 		Object response = responseContext.getEntity();
 		if (response == null) {
+			if (feature.getBoot().isNullDataResponseEnabled()) {
+				Document document = new Document();
+				document.setData(Nullable.nullValue());
+				responseContext.setEntity(document);
```

Comment:
The `setEntity` method is from JAX-RS. The new content is directly set on the `ContainerResponseContext` object passed to the `filter` method.

I will take care of the changes you mentioned below soon.